js-intro.js

```diff
@@ -36,16 +61,42 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
+    onlyVowels = (array) => {
+    var vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'];
+    return array
+    .split('')
+    .filter(char => !vowels.includes(char))
+    .join('');
 
+}
```

Comment:
Very concise solution, good job. If you'd like, you can append the `.split()` `.filter()` and `.join()` to your return line to save some lines of code.